ivadomed/loader/loader.py

```diff
@@ -162,13 +162,16 @@ def __init__(self, input_filenames, gt_filenames, metadata=None, slice_axis=2, c
         # list of GT for multiclass segmentation
         self.gt_handle = []
 
-        # Unlabeled data (inference time)
+        # Labeled data (ie not inference time)
         if self.gt_filenames is not None:
             if not isinstance(self.gt_filenames, list):
                 self.gt_filenames = [self.gt_filenames]
             for gt in self.gt_filenames:
                 if gt is not None:
-                    self.gt_handle.append(nib.load(gt))
+                    if isinstance(gt, str):  # this tissue has annotation from only one rater
+                        self.gt_handle.append(nib.load(gt))
+                    else:  # this tissue has annotation from several raters
+                        self.gt_handle.append([nib.load(gt_rater) for gt_rater in gt])
```

Comment:
Keep it as a single image if `target_suffix` was a string. Otherwise get a list of images (ie in case of multiple raters).